cmd/admin-bw-info.go

```diff
@@ -0,0 +1,392 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	json "github.com/minio/mc/pkg/colorjson"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/bucket/bandwidth"
+	"github.com/minio/minio/pkg/console"
+	"github.com/minio/minio/pkg/madmin"
+)
+
+var adminBwInfoCmd = cli.Command{
+	Name:   "bandwidth",
+	Usage:  "Show bandwidth info for buckets stored on the MinIO server",
+	Action: mainAdminBwInfo,
+	Before: setGlobalsFromContext,
+	Flags:  append(adminBwFlags, globalFlags...),
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Show the bandwidth usage for the MinIO server
+     {{.Prompt}} {{.HelpName}} --buckets bucket1 --buckets bucket2 play/
+  2. Show the bandwidth usage for a distributed MinIO servers setup
+     {{.Prompt}} {{.HelpName}} --buckets bucket1 --buckets bucket2 --servers minio01/ --servers minio02/ --servers minio03/ --servers minio04/
+`,
+}
+
+var adminBwFlags = []cli.Flag{
+	cli.StringSliceFlag{
+		Name:  "buckets",
+		Usage: "format '--buckets <bucket>', multiple values allowed for multiple buckets",
+	},
+	cli.StringSliceFlag{
+		Name:  "servers",
+		Usage: "format '--servers <server>', multiple values allowed for multiple servers for a distributed setup",
```

Comment:
The Admin API should be querying all the servers and the mc command only needs to talk to any one of the instances.